edx-solutions/xblock-drag-and-drop-v2
# Remove support for numerical input.

This PR removes support for numeric input feature. Nobody is using that feature anymore and it adds a lot of complexity to the codebase.

**JIRA tickets**: https://openedx.atlassian.net/browse/SOL-1870

**Dependencies**: None

**Sandbox LMS URL**: http://dndv2.opencraft.hosting/
**Sandbox Studio URL**: http://studio-dndv2.opencraft.hosting/

**Testing instructions**:

This PR does not add any new features and makes no user-visible changes (assuming users were not using the numeric input feature) so there is no specific functionality to test.
Testers should verify that DnDv2 continues to function correctly.

**Reviewers**
